pkgs/top-level/php-packages.nix

```diff
@@ -1,10 +1,12 @@
 { pkgs, php }:
 
-let self = with self; {
-  buildPecl = import ../build-support/build-pecl.nix {
-    inherit php;
-    inherit (pkgs) stdenv autoreconfHook fetchurl;
-  };
+let
+  self = with self; {
+    buildPecl = import ../build-support/build-pecl.nix {
+      inherit php;
+      inherit (pkgs) stdenv autoreconfHook fetchurl;
+    };
+  php7 = pkgs.lib.versionAtLeast php.version "7.0";
```

Comment:
To clarify my first question: Do we need to be able to access `phpPackages.isPhp7`, or can we just have this let-bound?